src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,24 +1,108 @@
 package core.basesyntax;
 
-/**
- * <p>Реалізувати свою HashMap, а саме методи `put(K key, V value)`, `getValue()` та `getSize()`.
- * Дотриматися основних вимог щодо реалізації мапи (initial capacity, load factor, resize...)
- * За бажанням можна реалізувати інші методи інтрефейсу Map.</p>
- */
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
 
+    private static class Entry<K, V> {
+        private K key;
+        private V value;
+        Entry<K, V> next;
+
+        private Entry(K key, V value) {
+            this.key = key;
+            this.value = value;
+        }
+
+    }
+
+    private Entry<K, V>[] table;
+    private int size;
+    private float loadFactor = 0.75f;
```

Comment:
@joej0e Please move 0.75f as a constant. Same for 16

Reply:
to show that this is a power of two